### DISCUSSION TITLE
redis sentinel configuration for vmq_diversity.redis.host

### DISCUSSION THREAD

**Discussion by vshirol** [vernemq/vernemq#1972] [Q&A]
Hi All,

We are using VMQ diversity module for lua plugin in VerneMQ along with redis. Its working fine when we use master/slave redis setup. Question is we are moving towards to redis sentinel setup (3 sentinel instances), so in this case what needs to be configured for vmq_diversity.redis.host and vmq_diversity.redis.port ? Does VerneMQ Version: 1.10.2 support connecting to sentinel instance if so are there any special configurations required and what needs to configured for vmq_diversity.redis.host, vmq_diversity.redis.port properties in vernemq.conf file.

Thanks & Regards,
Vasant

**Comment by ioolkos** [comment]
Redis Sentinel is not supported. It seems that Redis Sentinel unfortunately needs the client library to be able to ask what the current Redis master node is.

**Reply by vshirol** [reply]
Thanks @ioolkos for the information and your quick response. In that case I think for us one of the option would be make use sentinel proxy something like: https://github.com/RedisLabs/sentinel_tunnel.

**Reply by ioolkos** [reply]
Okay, if `sentinel_tunnel` should be a viable option, please let us know.

**Reply by vshirol** [reply]
Sure we will test and let you know here.

**Reply by vshirol** [reply]
@ioolkos We did a test with https://github.com/RedisLabs/sentinel_tunnel along with VerneMQ(1.12.4) for redis sentinel setup and it works fine.

**Reply by ioolkos** [reply]
@vshirol thanks for testing & letting us know!
